sdk/fabric/Azure.ResourceManager.Fabric/src/Generated/Models/RpSkuEnumerationForExistingResourceResult.Serialization.cs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Fabric.Models
+{
+    internal partial class RpSkuEnumerationForExistingResourceResult : IUtf8JsonSerializable, IJsonModel<RpSkuEnumerationForExistingResourceResult>
```

Comment:
uppercase `Rp`

Reply:
This is a paged result model which will never be public in the future, I think we do not have to change this